src/NuGet/Microsoft.CodeAnalysis.Remote.Workspaces.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.CodeAnalysis.Remote.Workspaces</id>
+    <summary>
+      A shared package used by the .NET Compiler Platform ("Roslyn") including support for coordinating analysis of projects and solutions using a separate server process.
+    </summary>
+    <description>
+      A shared package used by the .NET Compiler Platform ("Roslyn") including support for coordinating analysis of projects and solutions using a separate server process. Do not install this package manually, it will be added as a prerequisite by other packages that require it.
+
+Supported Platforms:
+- .NET Framework 4.5
+    </description>
+    <dependencies>
+      <dependency id="Microsoft.CodeAnalysis.Common" version="[$version$]" />
+      <dependency id="Microsoft.CodeAnalysis.Workspaces.Common" version="[$version$]" />
+      <dependency id="Microsoft.Composition" version="$MicrosoftCompositionVersion$" />
+    </dependencies>
+
+    <language>en-US</language>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <version>$version$</version>
+    <authors>$authors$</authors>
+    <licenseUrl>$licenseUrl$</licenseUrl>
+    <projectUrl>$projectUrl$</projectUrl>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <tags>$tags$</tags>
+    <serviceable>true</serviceable>
+  </metadata>
+  <files>
+    <file src="Microsoft.CodeAnalysis.Remote.Workspaces.dll" target="lib\portable-net45+win8" />
+    <file src="Microsoft.CodeAnalysis.Remote.Workspaces.xml" target="lib\portable-net45+win8" />
+    <file src="Microsoft.CodeAnalysis.Remote.Workspaces.dll" target="lib\net45" />
```

Comment:
I've copied the targets from Microsoft.CodeAnalysis.Workspaces.Common (they should be the same).